### PR TITLE
Clarify path for ccache

### PR DESCRIPTION
**Pull request by plaindocs**
Fix for #984 

@iriberri

### CODE REVIEW

File: user/caching.md [hunk 1]
```diff
@@ -134,17 +134,27 @@ caches `$HOME/.cache/pip`.
...
+If you are using `ccache`, use:
 
 ```yaml
 language: c # or other C/C++ variants
 
 cache: ccache
 ```
 
-caches `$HOME/.ccache`, and adds `/usr/lib/ccache` to the front of `$PATH`.
+to cache `$HOME/.ccache`, and add `/usr/lib/ccache` to the front of `$PATH`.
```
Comment (iriberri) [inline user/caching.md]: In the case of Linux, Travis automatically edits the PATH when it detects that ccache is being used: https://github.com/travis-ci/travis-build/blob/35b5046996ef1f50d9be2206e921666e0937b8d5/lib/travis/build/script/c.rb#L12-L13

Comment (plaindocs) [inline user/caching.md]: Yeah, but don't we already mention that? 

Ah, I see what you mean. Interesting take. I'll reword.

File: user/caching.md [hunk 2]
```diff
@@ -134,17 +134,27 @@ caches `$HOME/.cache/pip`.
...
 
-For caching `ccache` files, use:
+If you are using `ccache`, use:
 
 ```yaml
 language: c # or other C/C++ variants
 
 cache: ccache
-```
+``
```
Comment (iriberri) [inline user/caching.md]: Oops! Missing quote here! :) and I think that the `automatically` of the next line is also duplicated, right?

Comment (plaindocs) [inline user/caching.md]: :-( some keyboard gibberish. Fixed. Thanks for checking!

**Pull request merged**